packages/nimble-components/src/table-column/enum-text/tests/table-column-enum-text.stories.ts

```diff
@@ -0,0 +1,135 @@
+import { html, ref } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import {
+    createUserSelectedThemeStory,
+    incubatingWarning
+} from '../../../utilities/tests/storybook';
+import { tableTag } from '../../../table';
+import { tableColumnEnumTextTag } from '..';
+import {
+    SharedTableArgs,
+    sharedTableArgTypes,
+    sharedTableArgs
+} from '../../base/tests/table-column-stories-utils';
+import { tableColumnTextTag } from '../../text';
+import { mappingTextTag } from '../../../mapping/text';
+import { sharedMappingValidityDescription } from '../../enum-base/tests/shared';
+
+const simpleData = [
+    {
+        firstName: 'Ralph',
+        lastName: 'Wiggum',
+        status: 'fail',
+        isChild: true
+    },
+    {
+        firstName: 'Marge',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: false
+    },
+    {
+        firstName: 'Homer',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: false
+    },
+    {
+        firstName: 'Bart',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: true
+    },
+    {
+        firstName: 'Greg',
+        lastName: '',
+        status: 'foo',
+        isChild: true
+    },
+    {
+        firstName: 'Frank',
+        lastName: '',
+        status: 'bar',
+        isChild: false
+    }
+];
+
+const enumTextColumnDescription = 'The `nimble-table-column-enum-text` column renders string, number, or boolean values as mapped text in the `nimble-table`.';
+
+const metadata: Meta<EnumTextColumnTableArgs> = {
+    title: 'Incubating/Table Column - Enum Text',
+    parameters: {
+        docs: {
+            description: {
+                component: enumTextColumnDescription
+            }
+        }
+    }
+};
+
+export default metadata;
+
+interface EnumTextColumnTableArgs extends SharedTableArgs {
+    fieldName: string;
+    keyType: string;
+    checkValidity: () => void;
+    validity: () => void;
+}
+
+export const enumTextColumn: StoryObj<EnumTextColumnTableArgs> = {
+    // prettier-ignore
+    render: createUserSelectedThemeStory(html<EnumTextColumnTableArgs>`
+        ${incubatingWarning({ componentName: 'table', statusLink: 'https://github.com/orgs/ni/projects/7/views/21' })}
+        <${tableTag}
+            ${ref('tableRef')}
+            data-unused="${x => x.updateData(x)}"
+        >
+            <${tableColumnTextTag} field-name="firstName" >
+                Name
+            </${tableColumnTextTag}>
+            <${tableColumnEnumTextTag} 
```

Comment:
My (admittedly weak) justification is that you can the keys/mappings/key-type from the Code view of the Docs page. And (weaker still) if someone's looking at the story's source in the repo.

Reply:
I think it will look redundant for almost all users, so I'd vote for not adding them (and this goes for the icon column story that will be coming, too).